app/templates/adf-cli-acs-aps-template/src/app.config.json

```diff
@@ -2,23 +2,12 @@
   "$schema": "../node_modules/@alfresco/adf-core/app.config.schema.json",
   "ecmHost": "{protocol}//{hostname}{:port}",
   "bpmHost": "{protocol}//{hostname}{:port}",
+  "authType": "BASIC",
   "identityHost": "{protocol}//{hostname}{:port}/auth/realms/alfresco",
   "providers": "ALL",
   "application": {
     "name": "Alfresco ADF Application"
   },
-  "authType": "OAUTH",
-  "oauth2": {
```

Comment:
let's leave the ouath2 conf so if someone want change maybe is easy figure out what is needed

Reply:
Deal!